source/styles.json

```diff
@@ -0,0 +1,66 @@
+{
+  "rows": {
```

Comment:
I would use a more descriptive name here... 
Think in a bigger channel, where you might have other "rows" rather than the rows of the main "rowList".

Maybe "rowList.rows" would fit better

Reply:
Yes you're right. And I realized you can't scope using "."